docs/interfaces/http.rst

```diff
@@ -267,6 +267,8 @@ ID    Data Type
 ----- -------------------
 15    Unchecked Object
 ----- -------------------
+19    Time with time zone
```

Comment:
Maybe we should start describing the format as well?

Reply:
sure! for all types in the table?

Reply:
I guess for all to make it consistent. I'd also change the [table syntax to list](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table), so that it is easier to edit.